pages/inspector/inspector-debugger.mdx

```diff
@@ -82,6 +86,49 @@ The debugger provides controls for managing the live stream:
 - **Resume** - Resume the live stream to continue receiving events
 - **Reset** - Clear all events from the current view to start fresh. Useful when you are finding your stream session or want to start fresh.
 
+### Decrypting property values
+
+The Inspector Debugger supports property value validation, allowing you to see the actual values being sent with your events and verify they match your expected data. This is useful for catching [property value issues](/inspector/issue-types-in-inspector#property-value-issues) where values that don't match the [property constraints](/data-design/avo-tracking-plan/properties#property-types-and-constraints) defined in Avo, such as allowed values and regex patterns for string properties, min and max values for numerical properties, and required or unexpected keys in object properties.
+
+Property values are encrypted end-to-end for privacy and security. You can always see if there is an issue related to the value but to see the actual values you need to decrypted them using your private key.
+
+#### How encryption works
+
+Property values are encrypted end-to-end for privacy and security:
+
+- **Avo never stores actual values** - Only encrypted values are stored on Avo's servers
+- **Public/private key encryption** - You generate a key pair using the Avo CLI. Avo uses the public key to encrypt values, and only you can decrypt them with your private key
+- **Avo cannot decrypt your data** - The public key can only be used for encryption, not decryption
+
+![Inspector Debugger with encrypted values](/images/inspector/debugger/inspector-debugger-event-details.png)
+
+#### Setting up property value validation
+
+To enable property value validation:
+
+1. **Generate encryption keys** - Use the Avo CLI to generate a public/private key pair when setting up value validation in the Inspector SDK
+2. **Store your private key securely** - Keep your private key in a secure location like 1Password or another secrets manager. You'll need this key to decrypt values in the debugger
+3. **Configure the SDK** - Follow the [Inspector SDK setup guide](/inspector/sdk/avo-inspector-sdk-setup) to configure value validation with your public key
```

Comment:
You should not need to provide an encryption key to enable value validation, only to see the actual values